docs/fsharp/language-reference/async-expressions.md

```diff
@@ -1,11 +1,13 @@
 ---
-title: Asynchronous Workflows
-description: Learn about support in the F# programming language for performing computations asynchronously, which execute without blocking execution of other work.
-ms.date: 08/15/2020
+title: Async Expressions
+description: Learn about support in the F# programming language for writing async expressions, which execute without blocking execution of other work.
+ms.date: 10/29/2021
 ---
-# Asynchronous workflows
+# Async expressions
 
-This article describes support in F# for performing computations asynchronously, that is, without blocking execution of other work. For example, asynchronous computations can be used to write applications that have UIs that remain responsive to users as the application performs other work.
+This article describes support in F# for async expressions, one way of performing computations asynchronously, that is, without blocking execution of other work. For example, asynchronous computations can be used to write applications that have UIs that remain responsive to users as the application performs other work.
+
+Asynchronous code can also be authored using [task expressions](task-expressions.md), which create .NET tasks directly. Using task expressions is preferred when interoperating extensively with .NET libraries that create or consume .NET tasks. When writing most asynchronous code in F#, F# async expressions are preferred because they are more succinct, more compositional and avoid certain caveats associated with .NET tasks.
```

Comment:
s/more compositional and/more compositional, and/ 

Microsoft style uses Oxford comma.

Reply:
thanks